airflow/configuration.py

```diff
@@ -1531,9 +1531,9 @@ def initialize_config() -> AirflowConfigParser:
         if local_conf.getboolean("core", "unit_test_mode"):
             local_conf.load_test_config()
 
-    # Make it no longer a proxy variable, just set it to an actual string
+    # The conf.get should provide the default_config in case the value is not set in webserver configs
```

Comment:
```suggestion
```

Don't think we need this comment, that's always the behavior of `get`.

Reply:
Sure, made the changes